js-classes/inheritance-es5.js

```diff
@@ -0,0 +1,106 @@
+function Car(name, modal, year, color, maxSpeed, fuelCapacity, fuelConsumption){
+    
+    this.name = name;
+    this.modal = modal;
+    this.year = year;
+    this.color = color;
+    this.maxSpeed = maxSpeed;
+
+    if(!fuelCapacity){
+        this.fuelCapacity = 60;
+    }else{
+        this.fuelCapacity = fuelCapacity;
+    }
```

Comment:
This could be simplified:
```suggestion
  this.fuelCapacity = fuelCapacity || 60;
```